evmtool docker image publish failing

the buildDocker GHA failed on merge of [this PR 8304](https://github.com/hyperledger/besu/pull/8304) - and the [failure link](https://github.com/hyperledger/besu/actions/runs/13381142166/job/37369815247) - the error is
```
#7 4.118 E: Version '20190909*' for 'ca-certificates-java' was not found

ERROR: failed to solve: process "/bin/sh -c apt-get update $NO_PROXY_CACHE  &&   apt-get -o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0     --no-install-recommends -q --assume-yes install ca-certificates-java=20190909*  &&   apt-get -o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0     --no-install-recommends -q --assume-yes install openjdk-21-jre-headless=21*  &&   apt-get clean  &&   rm -rf /var/cache/apt/archives/* /var/cache/apt/archives/partial/*  &&   rm -rf /var/lib/apt/lists/*  &&   adduser --disabled-password --gecos \"\" --home /opt/besu besu  &&   chown besu:besu /opt/besu" did not complete successfully: exit code: 100

> Task :ethereum:evmtool:distDocker FAILED

FAILURE: Build failed with an exception.
```

--- a/ethereum/evmtool/src/main/docker/Dockerfile
+++ b/ethereum/evmtool/src/main/docker/Dockerfile
@@ -7,7 +7,7 @@ ENV NO_PROXY_CACHE="-o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true
 RUN apt-get update $NO_PROXY_CACHE  && \
   # $NO_PROXY_CACHE must not be used here or otherwise will trigger a hadolint error
   apt-get -o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0 \
-    --no-install-recommends -q --assume-yes install ca-certificates-java=20190909*  && \
+    --no-install-recommends -q --assume-yes install ca-certificates-java=20240618*  && \
   apt-get -o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0 \
     --no-install-recommends -q --assume-yes install openjdk-21-jre-headless=21*  && \
   # Clean apt cache  \

--- a/ethereum/evmtool/src/main/docker/Dockerfile
+++ b/ethereum/evmtool/src/main/docker/Dockerfile
@@ -7,9 +7,7 @@ ENV NO_PROXY_CACHE="-o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true
 RUN apt-get update $NO_PROXY_CACHE  && \
   # $NO_PROXY_CACHE must not be used here or otherwise will trigger a hadolint error
   apt-get -o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0 \
-    --no-install-recommends -q --assume-yes install ca-certificates-java=20240618*  && \
-  apt-get -o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0 \
-    --no-install-recommends -q --assume-yes install openjdk-21-jre-headless=21*  && \
+    --no-install-recommends -q --assume-yes install openjdk-21-jre-headless=21* adduser=3* && \
   # Clean apt cache  \
   apt-get clean  && \
   rm -rf /var/cache/apt/archives/* /var/cache/apt/archives/partial/*  && \

